Api/Webhook/Webhook.php

```diff
@@ -283,13 +283,14 @@ private function isRefund(array $event): bool
     private function addEventLog(string $action, string $status): void
     {
         $eventLog = $this->eventLogFactory->create();
+        $time = $this->timezone->date(new \DateTime());
         $eventLog->addData([
             'request_route' => $this->request->getPathInfo(),
             'request_body' => $this->request->getContent(),
             'request_method' => $this->request->getMethod(),
             'request_action' => $action,
             'response_code' => $status,
-            'date' => $this->timezone->date()->getTimestamp()
+            'date' => $time->format('Y-m-d H:i:s')
```

Comment:
does this go ok with timezone? because you are omitting them here (same question for other places)

Reply:
It is after format

Reply:
But do they store it UTC or any timezone it happens to loaded at the moment.